modules/calcite/src/test/java/org/apache/ignite/internal/processors/query/calcite/rules/ProjectScanMergeRuleTest.java

```diff
@@ -122,7 +121,7 @@ public void testProjects() {
             .check();
 
         checkQuery("SELECT NAME FROM products d WHERE CAT_ID > 1;")
-            .matches(containsIndexScan("PUBLIC", "PRODUCTS"))
+            .matches(containsAnyScan("PUBLIC", "PRODUCTS"))
```

Comment:
i broke this tests and i hope no need to fix them in a such a way, if we have possibility to obtain here TasbleScan - seems we have a problem )

Reply:
This particular test class is intended to verify ProjectScanMergeRule. See no reason to verify the index choosing, especially on the data set of 4 rows.